src/sgp4_sys.rs

```diff
@@ -472,6 +486,88 @@ pub fn run_sgp4(
     }
 }
 
+#[derive(Debug)]
+pub struct ClassicalOrbitalElements {
+    pub p: c_double,       // semilatus rectum               km
+    pub a: c_double,       // semimajor axis                 km
+    pub ecc: c_double,     // eccentricity
+    pub incl: c_double,    // inclination                    0.0  to pi rad
+    pub omega: c_double,   // longitude of ascending node    0.0  to 2pi rad
+    pub argp: c_double,    // argument of perigee            0.0  to 2pi rad
+    pub nu: c_double,      // true anomaly                   0.0  to 2pi rad
+    pub m: c_double,       // mean anomaly                   0.0  to 2pi rad
+    pub arglat: c_double,  // argument of latitude      (ci) 0.0  to 2pi rad
+    pub truelon: c_double, // true longitude            (ce) 0.0  to 2pi rad
+    pub lonper: c_double,  // longitude of periapsis    (ee) 0.0  to 2pi rad
```

Comment:
if this is the Rust interface we should probably use `f64` here and handle the conversion at FFI invocation time

Reply:
This module is all FFI; the actual Rust interface is `StateVector`.